DASH-IF-IOP.bs.md

```diff
@@ -10,8 +10,500 @@ Live discussion in #document-authoring on Slack.
+
+All periods SHALL be consecutive and non-overlapping. A period MAY have a duration of zero.
+
+Note: A period with a duration of zero might, for example, be the result of ad-insertion logic deciding not to insert any ad.
+
+<div class="example">
+The below MPD example consists of two 20-second periods. The duration of the first period is calculated using the start point of the second period.
+
+<xmp highlight="xml">
+<MPD type="static" mediaPresentationDuration="PT40S">
+	<Period>
+		...
+	</Period>
+	<Period start="PT20S" duration="PT20S">
+		...
+	</Period>
+</MPD>
+</xmp>
+</div>
+
+In a static MPD, the first period SHALL start at the zero point of the [=MPD timeline=]. In a dynamic MPD, the first period SHALL start at or after the zero point of the [=MPD timeline=].
+
+In a static MPD, the last period SHALL have a `Period@duration`. In a dynamic MPD, the last period MAY lack a `Period@duration`, in which case it SHALL be considered to have an unlimited duration.
+
+In a static MPD, `MPD@mediaPresentationDuration` SHALL be present. In a dynamic MPD, `MPD@mediaPresentationDuration` SHALL be present if the content of the MPD will no longer be updated and SHALL NOT be present if the content of the MPD might be updated.
+
+Note: Publishing a dynamic MPD that will not be updated enables a service to schedule the availability of content that has already been fully generated (e.g. a finished live event or a scheduled playback of existing content).
+
+When present, `MPD@mediaPresentationDuration` SHALL accurately indicate the duration between the zero point on the [=MPD timeline=] and the end of the last [=period=].
+
+## Representations ## {#timing-representation}
+
+A <dfn>representation</dfn> is a sequence of references to [=media segments=] containing media samples. Each representation belongs to exactly one [=adaptation set=] and to exactly one [=period=], although [[#timing-connectivity|a representation may be connected with a representation in another period]].
+
+A reference to a [=media segment=] determines which [=media segment=] corresponds to which time span on the [=MPD timeline=]. The exact mechanism used to define references depends on the [=addressing mode=] used by the representation. All representations in the same [=adaptation set=] SHALL use the same [=addressing mode=].
+
+A representation SHALL reference a set of media segments that ensures the [=MPD timeline=] is covered with segments at least from the beginning of the period active range to the end of the period active range.
+
+In a static MPD, the period active range SHALL be the entire time span of a [=period=].
+
+<figure>
+	<img src="Images/Timing/StaticMpdMustBeCovered.png" />
+	<figcaption>In a static MPD, the entire period must be covered with [=media segments=].</figcaption>
```

Comment:
I am intentionally trying to avoid promoting the "exact match" cases in this document because I consider an exact match to be a special case. The usual case I consider to be a case where there is no alignment between periods/segments/samples.

Reply:
Makes sense!